script.js

```diff
@@ -1,14 +1,29 @@
 function shifumi() {
-    const user_choice = ['rock', 'paper', 'scissors']
-    const choice = prompt(`Choose one of the following: ${user_choice.join(', ')}.`)
+    const user_choice = ['rock', 'paper', 'scissors'];
+    const choice = prompt(`Choose one of the following: ${user_choice.join(', ')}.`);
 
     if (!user_choice.includes(choice)) {
         alert(`Attention, ${choice} is not one of the ${user_choice.length} choices`);
         return shifumi()
     }
 
     alert(`Congratulations, you chose ${choice}`);
-}
 
+    //AI choice
+    const ia = Math.floor(Math.random() * user_choice.length)
+    const ia_choice = user_choice[ia]
+    alert(`AI chose ${ia_choice}, let's play !`);
+
+    //who wins
+    if (ia_choice.includes(choice)) {
+        alert(`there is an equality here !`)
+        shifumi();
+    } else if ((ia_choice == user_choice[0] && choice == user_choice[1]) || (ia_choice == user_choice[1] && choice == user_choice[2]) || (ia_choice == user_choice[2] && choice == user_choice[0])) {
```

Comment:
nit: here this is a bit difficult to read: 
- We can factorize it, but we will see it later
- You are checking the strings here, maybe you could check the ids instead?